texar/tf/modules/decoders/gpt2_decoder.py

```diff
@@ -0,0 +1,317 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+GPT2 decoders.
+"""
+
+import tensorflow as tf
+
+from texar.tf.modules.decoders.transformer_decoders import TransformerDecoder
+from texar.tf.modules.embedders import PositionEmbedder, WordEmbedder
+from texar.tf.modules.pretrained.gpt2 import PretrainedGPT2Mixin
+
+
+__all__ = [
+    "GPT2Decoder",
+]
+
+
+class GPT2Decoder(PretrainedGPT2Mixin):
```

Comment:
This differs from Texar-Pytorch by not inheriting `TransformerDecoder`. What's the reason?